esrally/utils/sysstats.py

```diff
@@ -53,6 +53,16 @@ def cpu_model():
     return "Unknown"
 
 
+def cpu_arch():
```

Comment:
How about we implement this as below? This has the advantage that it is part of the Python standard library.

```python
def cpu_arch():
    return platform.uname().machine
```